appveyor.yml

```diff
@@ -2,10 +2,6 @@ version: 6.0.0-alpha.18-{build}
 
 image: Visual Studio 2015
 
-cache:
-  - '%LocalAppData%\Microsoft\dotnet'
-  - '%HOMEDRIVE%%HOMEPATH%\.nuget\packages'
```

Comment:
There is also a language to describe when to clear the cache https://www.appveyor.com/docs/build-cache/#cache-dependencies